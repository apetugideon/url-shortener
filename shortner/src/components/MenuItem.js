import React from "react";

function MenuItem(props) {
    let iconClass = props.iconClass;
    return (
        <span className="top-menus text-right" style={{color: props.color || ""}}>
            <i className={iconClass}></i>
            &nbsp;{props.menuDesc}
        </span>
    );
}

export default MenuItem;