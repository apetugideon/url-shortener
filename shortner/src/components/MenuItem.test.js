import React from 'react';
import { shallow, mount } from 'enzyme';
import MenuItem from './MenuItem';


describe('MenuItem component test', () => {
   test('renders without crashing', () => {
      let menuItem = mount(<MenuItem />);
      expect(menuItem).toMatchSnapshot();
    });
});
