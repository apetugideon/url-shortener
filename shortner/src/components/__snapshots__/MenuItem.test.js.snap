// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem component test renders without crashing 1`] = `ReactWrapper {}`;
